packages/@ama-sdk/showcase-sdk/openapitools.json

```diff
@@ -2,7 +2,7 @@
   "$schema": "https://raw.githubusercontent.com/OpenAPITools/openapi-generator-cli/master/apps/generator-cli/src/config.schema.json",
   "spaces": 2,
   "generator-cli": {
-    "version": "6.6.0",
+    "version": "7.4.0",
```

Comment:
Shouldn't we have an ng-update to change this value on already generated sdk ?

Reply:
I will do it in another PR :)